build_projects/update-dependencies/update-dependencies.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.DotNet.VersionTools" Version="$(VersionToolsVersion)" />
+    <PackageReference Include="NuGet.Packaging" Version="$(NuGetPackagingPackageVersion)" />
```

Comment:
This shouldn't be needed.

I've had trouble in the past with referencing NuGet from here.  We can get into mis-match situations where VersionTools needs NuGet version X and CLI needs version Y, and they don't work together.

It is better to just use the NuGet that is referenced by VersionTools here.

Reply:
I had to pin the NuGet version because I was hitting a failure to load dll error otherwise.

@dagood can you let me know when there is a build available with your fix and what the version number is? I can try using that and removing the pinning above.

Reply:
Give `2.1.0-prerelease-02430-04` a try.